relay-server/src/metrics.rs

```diff
@@ -0,0 +1,200 @@
+use relay_common::metrics::{CounterMetric, HistogramMetric, SetMetric, TimerMetric};
+
+/// Set metrics used by Relay
+pub enum RelaySets {
+    /// Represents the number of active projects in the current slice of time
+    UniqueProjects,
+}
+
+impl SetMetric for RelaySets {
+    fn name(&self) -> &'static str {
+        match self {
+            RelaySets::UniqueProjects => "unique_projects",
+        }
+    }
+}
+
+/// Histogram metrics used by Relay.
+pub enum RelayHistograms {
+    /// The number of events in the queue as a percentage of the maximum number of events
+    /// that can be stored in the queue ( 0 ... the queue is empty, 1 ... the queue is full
+    /// and no additional events can be added).
+    EventQueueSizePct,
+    /// The number of events in the queue at the sampling moment.
```

Comment:
_nit_: Remove "at the sampling moment". This is true for every metric.

Might be worth explaining here what the queue is (and refering to it from other queue metrics):
Events are queued once they have been accepted by the web endpoint. Relay fetches information on the Project, processes the event and then forwards it to the upstream. After that, the event is dequeued.